app/assets/javascripts/code/services/_sensors.js

```diff
@@ -58,7 +50,7 @@ export const sensors = (params, $http) => {
         console.log('initSelected() - sensorId is NOT null')
       }
       this.selectedParameter = self.findParameterForSensor(self.selected());
-      this.availableSensors = self.findAvailableSensorsForParameter(self.selectedParameter);
+      this.availableSensors = findAvailableSensorsForParameter(sort, this.sensors, self.selectedParameter);
```

Comment:
Why do we sometimes use this and later self? `this.sensors, self.selectedParameter`. Could be just `this` everywhere in this file?

Reply:
We prolly could but I'm leaving that for another round. Good catch nonetheless